src/v1/session.js

```diff
@@ -65,6 +65,9 @@ class Session {
       statement = statement.text;
     }
     assertString(statement, 'Cypher statement');
+    if (statement.length == 0) {
+      throw new TypeError('Cypher statement is expected to be a non-empty string.');
```

Comment:
Maybe worth adding a `assertCypherStatement()` function in `util.js` that will verify given cypher is a non-empty string? Just to avoid minor code duplication